### ISSUE TITLE
Bug: relative path to /assets - abort(RuntimeError: abort(both async and sync fetching of the wasm failed)

### ISSUE THREAD

**Issue by sh1kxrv** [jepiqueau/jeep-sqlite#12]
Here is the diff that solved my problem:

```diff
diff --git a/node_modules/jeep-sqlite/dist/components/jeep-sqlite.js b/node_modules/jeep-sqlite/dist/components/jeep-sqlite.js
index 8528b4c..a33e855 100644
--- a/node_modules/jeep-sqlite/dist/components/jeep-sqlite.js
+++ b/node_modules/jeep-sqlite/dist/components/jeep-sqlite.js
@@ -4957,11 +4957,12 @@ class Database {
     this.vUpgDict = upgDict;
     this._isDBOpen = false;
     this.autoSave = autoSave;
+
   }
   async open() {
     try {
       const SQL = await sqlWasm({
-        locateFile: file => `assets/${file}`
+        locateFile: file => `/assets/${file}`
       });
       // retrieve the database if stored on localforage
       const retDB = await getDBFromStore(this.dbName, this.store);
```

**Comment by jepiqueau** [comment]
@xafxafxaf fixed in 1.5.5 Please close it after test

**Comment by sh1kxrv** [comment]
ty
